settings/language-vala.cson

```diff
@@ -10,3 +10,4 @@
       'stdout'
       'stderr'
     ]
+    'commentStart': '// '
```

Comment:
This has already been merged as a separate merge request, and has nothing to do with errordomain not being treated as a keyword. Please remove this change from this changeset.